src/include/security/helps.rb

```diff
@@ -352,9 +331,48 @@ def initialize_security_helps(include_target)
         "EXTRA_SERVICES"                            => _(
           "<P>Every running service is a potential target of a security attack. Therefore it is recommended to turn off all services which are not used by the system.</P>"
         )
-      } 
+      }
+
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
+
+      # Boot dialog help 2/4, diferent message based on default
+      if ::Security::CtrlAltDelConfig.default == "reboot"
+        help += _(
```

Comment:
We should probably split the message to the constant part and a variable part to save some work for the translators.

```ruby
if ::Security::CtrlAltDelConfig.default == "reboot"
  # TRANSLATORS: part of help text - default action (the default is reboot)
  details = _("Usually the system reboots...")
else
  # TRANSLATORS: part of help text - default action (the default is halt)
  details = _("By default the system...")
end

# TRANSLATORS: %s is help text - default action
help += _("<p><b>Interpretation of... %s</p>") % details
```